spec/q-spec.js

```diff
@@ -2491,6 +2491,27 @@ describe("node support", function () {
 
 });
 
+describe("browser support", function () {
+    var _Q;
+
+    beforeEach(function() {
+        _Q = Q;
+    });
+
+    afterEach(function() {
+        Q = _Q;
+    });
+
+    it("sets the global Q object to its original value", function() {
+        Q.noConflict();
+
+        // In this context the original value of Q is undefined.
+        if(typeof window !== 'undefined') {
```

Comment:
Attend to the prevailing style with regard to spaces. I believe `npm run lint` will inform, but if it does not, take care to place spaces between keywords and parens to distinguish such cases from function calls.